include/RAJA/policy/openmp/nested.hpp

```diff
@@ -0,0 +1,196 @@
+/*!
+ ******************************************************************************
+ *
+ * \file
+ *
+ * \brief   RAJA header file containing constructs used to run forallN
+ *
+ ******************************************************************************
+ */
+
+#ifndef RAJA_policy_openmp_nested_HPP
+#define RAJA_policy_openmp_nested_HPP
+
+#include "RAJA/config.hpp"
+#include "RAJA/pattern/nested.hpp"
```

Comment:
@artv3 please move header inclusions, other than RAJA/config.hpp, inside the RAJA_ENABLE_OPENMP guard.  Also, replace the copyright stuff with the short form we are now using (see openmp/forall.hpp to see what I mean).